config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
   scope '/eholdings' do
+    resource :health, only: [:show]
```

Comment:
Should this `'health` endpoint be scoped under `/eholdings` or non-scoped to just be`/health` so from within a cluster if you want to hit this module from its IP for example would it then be `34.123.121.34/eholdings/health` or should it be `34.123.121.34/health`?

Reply:
That's a good question. From the initial conversation the example given was `/admin/health`, so I assumed it'd be scoped to `/eholdings`. However, I just search through several `mod-*` folio modules and only `mod-notes` has a `/health` endpoint at all. And it's `/admin/health`. Since `/admin` isn't defined in the raml for `mod-notes` I don't think the scoping on this endpoint matters at all.

We better ask Hongwei what he'd like to see since the health endpoint was his suggestion

Reply:
@elrickvm after some more research, it looks like `mod-notes` and `mod-notify` reference `/admin/health` in their tests. Searching up the grapevine, `raml-module-builder` implements the `/admin` endpoints that apparently all of their raml-based modules inherit.

I'm going to adjust this to also be `/admin/health` so that it's in-line with what's expected of the other modules.

Reply:
@wwilsman makes sense if they use it for health checking the a pod. Seems like this should be implemented at the container level in the Dockerfile or like us in the Kubernetes yaml. Reason i say that is because this doesn't really give them the granular feedback of whats actually happening with the container but lets just align with whatever they are doing for sure. Yeah we can double check with Hongwei to make sure.

Reply:
@elrickvm Relevant part of the `raml-module-builder`'s readme: https://github.com/folio-org/raml-module-builder/#monitoring